app/lib/widgets/base_screens/base_list_screen.dart

```diff
@@ -58,7 +60,7 @@ class _BaseListScreenState extends State<BaseListScreen> {
 
     return Scaffold(
       appBar: AppBar(
```

Comment:
What do you think about adding the square logo as `leading`? (I'm sure we have a png in a fitting size ;)

Reply:
I think the logo might get in the way of the back button